src/module/widget.ts

```diff
@@ -107,3 +152,7 @@ export default class Widget extends Application {
     this.input.placeholder = localize(`Widget.Placeholder${n}`);
   }
 }
+function getCommandSchemaWithoutArguments(command: Command) {
+  const argumentStart = command.schema.indexOf(' ');
+  return command.schema.substring(0, argumentStart > 0 ? argumentStart : command.schema.length);
+}
```

Comment:
this could be a call to the CommandHandler function too but I don't know if its worth it to expose. To be reviewed later.